apps/funding-service/hardhat.config.ts

```diff
@@ -1,2 +1,14 @@
 import "@nomiclabs/hardhat-ethers";
 import "@nomicfoundation/hardhat-chai-matchers";
+// import { HardhatUserConfig } from "hardhat/types";
```

Comment:
is this still needed?

Reply:
will remove since we are not deploying a contract here or anything that allows a user to run funding service with hardhat. Even if hardhat is on chain id 1 like in sandbox we still can not use funding service completely because we would need a contract address.